statistics/handle/dump.go

```diff
@@ -114,6 +118,61 @@ func dumpJSONCol(hist *statistics.Histogram, CMSketch *statistics.CMSketch, topn
 	return jsonCol
 }
 
+// DumpStatsToJSONBlocks dumps statistic to json, then compresses the json to blocks by gzip.
+func (h *Handle) DumpStatsToJSONBlocks(dbName string, tableInfo *model.TableInfo, blockSize int) ([][]byte, uint64, error) {
+	js, err := h.DumpStatsToJSON(dbName, tableInfo, nil)
+	version := uint64(0)
+	for _, value := range js.Columns {
```

Comment:
We should check error first, otherwise `js` may be a nil pointer and cause some panic here.